vars/main.yml

```diff
@@ -1,3 +1,4 @@
 # vars file for pycharm
 ---
 pycharm_downloads_path: /var/lib/ansible/pycharm/downloads
+pycharm_download_mirror: 'http://download.jetbrains.com/python'
```

Comment:
If you want to be able to override this it should be in `defaults/main.yml`

Reply:
i have force-pushed a change, moving the default download mirror to defaults. I hope it was the "correct" workflow, because now the referenced commit under "view changes" is wrong.

Thank you!